### PR TITLE
ADEN-3257 SP for logged users

### PR DESCRIPTION
**Pull request by jacekjursza**
Disable AdBlock content control for logged in users

### CODE REVIEW

File: extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache [hunk 1]
```diff
@@ -4,11 +4,16 @@ window._sp_ = {
     config: {
         enable_rid: true,
         content_control_callback: function() {
-            var msg = '<h2>Ad blocker interference detected!</h2>';
+            var logGroup = 'ext.wikia.aRecoveryEngine.recovery.bootstrap',
+                msg = '<h2>Ad blocker interference detected!</h2>';
```
Comment (drets) [inline extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache]: What about making the same message [here](https://github.com/Wikia/app/blob/155c0f83b95b1e26c7e21796ce1fe6af347087ba/extensions/wikia/ARecoveryEngine/css/recoveryLock.scss#L7)?

File: extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache [hunk 2]
```diff
@@ -4,11 +4,16 @@ window._sp_ = {
...
                 msg += '<h3>If you have added a new rule to your ad blocker that interferes with the loading of ads on your pages, ';
                 msg += 'this Interference Screen is the expected behaviour. <br />';
                 msg += 'Removing the rule(s) should load the page as expected.</h3>';
```
Comment (drets) [inline extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache:11]: What about making the same message [here](https://github.com/Wikia/app/blob/155c0f83b95b1e26c7e21796ce1fe6af347087ba/extensions/wikia/ARecoveryEngine/css/recoveryLock.scss#L7) as part of this PR?

Comment (jacekjursza) [inline extensions/wikia/ARecoveryEngine/templates/ARecoveryEngineApiController_getBootstrap.mustache:11]: I agree that there should be one message, but. There is an issue with that. Basically, we can not generate HTML tags trough CSS. IMO we should wait till we get a real message from ComDev and then think how to implement it in both places. I wouldn't touch it now to not duplicate work.

**Comment by fraszczakszymon** [comment]
Please remove lock css for logged in users as well:
https://github.com/Wikia/app/blob/dev/extensions/wikia/ARecoveryEngine/ARecoveryEngineHooks.class.php#L23

**Comment by fraszczakszymon** [comment]
One more thing, we are recovering only TOP_LEADERBOARD and TOP_RIGHT_BOXAD. Should we add HOME_ / CORP_ slots?
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/provider/directGpt.js#L57

**Comment by fraszczakszymon** [comment]
LGTM 👍

**Pull request merged**